dietpi/dietpi-software

```diff
@@ -10385,7 +10392,27 @@ _EOF_
 			fi
 
 		fi
+#---------------------------------
+		software_id=142 # microk8s
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Installing
+
+			# APT deps
+			G_AGI snapd
+		#install
+		G_EXEC snap install microk8s --classic
```

Comment:
```suggestion
		G_EXEC_OUTPUT=1 G_EXEC snap install microk8s --classic
```
Since this can take a while that users can follow the process. It spams log files, since it prints process bar evolution and such over the same line, while in the log file every single percent creates a new line. But it's worth it for interactive installs, I'd say.